content/Guides/DistributedComplianceLedger.md

```diff
@@ -0,0 +1,142 @@
+| Vendor               | It can write vendor and product information to the ledger. Alliance members can have one or more vendor accounts.                                    |
+| Vendor Admin         | It can add and update the Vendor information table of the ledger.                                                                                    |
+| Certification Center | It can submit, update, delete or revoke certification status of a product based on the Certification application in the Certification Tool.          |
+
+### Main-Net and Test-Net
+
+| DCL Deployment       | Description                                                                                                                                                |
+| -------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Main-Net             |  It is the DCL utilized by the public and is regarded as the production environment. Transactions on the Main-Net involve real products and data.          |
+| Test-Net             |  It is used for testing and development purposes. It provides a safe environment for experimentation and troubleshooting before deploying to the Main-Net. |
+
+### Access and interaction with the DCL
+
+-   **Write** access to the ledger is restricted to approved accounts in the DCL.
+
+-   **Read** access is broadly available through observer nodes.
+
+### Getting an account for writing to the DCL
+
+-   Enrollment for Main-Net DCL accounts is managed through the "DCL Account(s)" section in the Certification Tool system (Knack). You can request access to the Certification Tool 
+    by sending an email to <help@csa-iot.org>.
+
+> Note: The Certification Tool portal will depend on the Membership type.
+
+-   Enrollment for Test-Net DCL accounts can be requested by sending an email to <dcl-admin@csa-iot.org>
+
+Review the following document for instructions: [HowTo - Writing to the DCL.pdf](https://groups.csa-iot.org/wg/members-all/document/27881)
+
+### Interacting with the DCL
+
+There are different options available for users and/or systems to interact with the DCL:
+
+-   Web User Interface, for intuitive navigation.
+-   API, for seamless integration.
+-   Command Line Interface (CLI), for efficient management.
+
+#### Web User Interface
+The Web UI allows users to interact with the DCL through a visual interface, making tasks more intuitive and reducing the likelihood of errors. There is no need to install any 
+application client or configuration setup.
+
+#### Command Line Interface
+
+The DCL CLI software (dcld) allows users to interact with the DCL via command line using a connection to a specific available ON. The CLI software is available at the following link: 

```

Comment:
I'd recommend using "observer node" instead of ON here, at first I thought it meant the word "on".

Also for "dcld" I would make it code font, to reinforce it's a CLI

Reply:
Done